packages/runtime/src/extensibility/facades/transport/IdentityRecoveryKitsFacade.ts

```diff
@@ -0,0 +1,19 @@
+import { Result } from "@js-soft/ts-utils";
+import { Inject } from "@nmshd/typescript-ioc";
+import { TokenDTO } from "../../../types";
+import { CreateIdentityRecoveryKitRequest, CreateIdentityRecoveryKitUseCase, DoesIdentityRecoveryKitExistResponse, DoesIdentityRecoveryKitExistUseCase } from "../../../useCases";
+
+export class IdentityRecoveryKitsFacade {
+    public constructor(
+        @Inject private readonly createIdentityRecoveryKitUseCase: CreateIdentityRecoveryKitUseCase,
+        @Inject private readonly existsIdentityRecoveryKitUseCase: DoesIdentityRecoveryKitExistUseCase
```

Comment:
Inconsistency.

Also, I'm still not completely happy with the new naming. I'd prefer the name of the use case to **describe** what the use case does instead of **asking a question**. That's why I suggested something starting with `check`, e.g. `checkForIdentityRecoveryKit`, `checkForExistingIdentityRecoveryKit`, `checkIdentityRecoveryKitExistence`, `checkIfIdentityRecoveryKitExists` ...